sites/site-utilities/README.md

```diff
@@ -0,0 +1,3 @@
+# Description
+
+This is a set of utilities to be used in the Creator and Component Explorer sites as they share similar logic.
```

Comment:
I may suggest something more concise:  "A set of utilities shared between the Creator and Component Explorer sites."